fetch.bs

```diff
@@ -5938,6 +5938,25 @@ initially a new {{AbortSignal}} object.
 
 <hr>
 
+<p>To
+<dfn export lt="create a Request object|creating a Request object">create a <code>Request</code> object</dfn>,
+given a <a for=/>request</a> <var>request</var> and header guard <var>guard</var>, run these steps:
+<!-- Still need to deal with Realms; maybe create AbortSignal explicitly? -->
```

Comment:
I think it's fine for the realm to be a parameter. I don't know if there's a way to determine a sensible default.

Reply:
See https://github.com/whatwg/fetch/issues/777#issuecomment-416225862 onward. I haven't really been able to make progress on Realm stuff. Not enough people seem to care...